src/Dialogs/SettingsDialog.vala

```diff
@@ -105,6 +105,10 @@ public class Akira.Dialogs.SettingsDialog : Gtk.Dialog {
         symbolic_switch = new SettingsSwitch ("use-symbolic");
         grid.attach (symbolic_switch, 1, 4, 1, 1);
 
+        grid.attach (new SettingsLabel (_("Invert Sidebars Order:")), 0, 5, 1, 1);
+        symbolic_switch = new SettingsSwitch ("invert-sidebar");
+        grid.attach (symbolic_switch, 1, 5, 1, 1);
+
```

Comment:
This shouldn't be here but underneath the `Interface` header, right after the Dark Theme switch.

Since this option requires a reboot, we should specify it in a helper text underneath.
Something like: "Close and reopen the application to apply this change."

The helper text should have a `dim-label` class and be aligned underneath the switch, like in the elementary OS Mouse Settings page.

Reply:
I added the label and tried (I'm not an elementary user) to copy elementary OS layout. The result seems rater strange:
![image](https://user-images.githubusercontent.com/33655232/87233715-364c4680-c3ca-11ea-9b91-68c07dc5cebd.png)

Maybe the white space that is present at the left is too big.

Reply:
That's okay for now. That panel needs some deep love and UI refresh, but we will do that later during the beta cycle since it's not a priority for now